README.md

```diff
@@ -0,0 +1,143 @@
+# Juvix
+
+<a href="https://github.com/anoma/juvix"><img align="right" width="300" alt="Tara the Juvix mascot" src="assets/images/tara-seating.svg" /></a>
+
+<!-- prettier-ignore -->
+| CI Status |
+| --------- |
+|<a href="https://github.com/anoma/juvix/actions/workflows/ci.yml"><img alt="CI status" src="https://github.com/anoma/juvix/actions/workflows/ci.yml/badge.svg" /> </a> |
+|<a href="https://github.com/anoma/juvix/actions/workflows/pages/pages-build-deployment"><img src="https://github.com/anoma/juvix/actions/workflows/pages/pages-build-deployment/badge.svg" alt="pages-build-deployment" /></a>|
+
+<!-- prettier-ignore -->
+| Codebase |
+| -------- |
+|<a href="https://github.com/anoma/juvix/tags"><img src="https://img.shields.io/github/v/release/anoma/juvix?include_prereleases"/></a>|
+|<a href="https://github.com/anoma/juvix/blob/main/LICENSE"><img src="https://img.shields.io/badge/license-GPL--3.0--only-blue.svg"/></a> |
+|<a href="https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=102404734&machine=standardLinux32gb&location=WestEurope"><img height="20pt" alt="Open the Juvix Standard Lib in Github Codespace" src="https://github.com/codespaces/badge.svg"/> </a>|
+
+Juvix is an open-source, constantly evolving functional programming language
+designed for writing privacy-preserving decentralized applications. Using Juvix, developers can write high-level programs which can be compiled to WASM directly, or through [VampIR][vampir] to circuits for private execution with [Taiga][taiga] on [Anoma][anoma] or Ethereum.
+
+## Getting Started
+
+To get started with Juvix, head over to the [documentation website][juvix-book]
+to learn more about the language and its features. You can also find
+installation instructions and tutorials to help you get started with writing
+Juvix programs. You can download the [latest release][latest-release] from the
+Juvix GitHub repository or use the web-based development environment, [Juvix
+Github Codespace][repo-codespace], which provides a pre-configured workspace
+ready to use with Juvix and the Haskell toolchain installed.
+
+## The Juvix programming language
+
+Juvix provides a high degree of assurance. The Juvix compiler runs several
+static analyses which guarantee the absence of runtime errors. Analyses
+performed include termination and type checking. As a result, functional
+programs, especially validity predicates, can be written with greater confidence
+in their correctness.
+
+Some language features in Juvix include:
+
+- Haskell/Agda-like syntax with support for Unicode
+- Type inference
+- Parametric polymorphism
+- User defined inductive data types
+- Higher-order functions
```

Comment:
I think it would be good to mention that Juvix is pure, perhaps by adding "Referential transparency" to the feature list:

```suggestion
- Higher-order functions
- Referential transparency
```